update.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
+
+msg () {
+	1>&2 echo $*
+}
+
+# check that we are in a prototype
+if ! grep -q govuk-prototype-kit package.json 2> /dev/null; then
+	msg 'ERROR you must run update.sh in a folder containing a GOV.UK Prototype Kit installation'
+	msg 'Exiting'
+	exit 1
+fi
+
+# make the update folder
+mkdir -p update
+# stop git from committing the update folder
+echo '*' > update/.gitignore
+
+cd update
+
+# download the latest Prototype Kit zip file
+if ! ls govuk-prototype-kit*.zip > /dev/null 2>&1; then
+	msg 'Downloading latest version of GOV.UK Prototye Kit...'
+	curl -LJO https://govuk-prototype-kit.herokuapp.com/docs/download
+	msg 'Done'
+fi
+
+# choose the zip file with the largest version number, use this to update from
+ZIP_FILE=$(ls govuk-prototype-kit*.zip | sort -V | tail -n1)
+ZIP_ROOT=${ZIP_FILE//.zip}
+OLD_VERSION=$(cat ../VERSION.txt)
+NEW_VERSION=${ZIP_ROOT#govuk-prototype-kit-}
+
+# unzip the zip file into the 'update' folder
+if [ ! -d "$ZIP_ROOT" ]; then
+	msg -n 'Extracting latest files and folders into update folder...'
+	unzip -q "$ZIP_FILE"
+	msg 'Done'
+fi
+
+cd ..
```

Comment:
we could probably put this into a sub-shell too, then the `cd`s don't need to be undone?

Reply:
Good thinking

Reply:
Ah, subshell doesn't work, because we're setting a bunch of variables :(